### PR TITLE
Rails 6.0 in Canary

### PR DESCRIPTION
**Pull request by yuenmichelle1**
Update kubernetes template replicas to 1 for staging canary to test rails 6.0 on canary

Describe your change here.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Review by camallen — APPROVED** [review]
:shipit:  

TBH this type of changes doesn't need a review from a maintainer of the repo. Should we relax the branch restrictions for cases like this to speed up the dev cycle?

**Comment by yuenmichelle1** [comment]
> :shipit:
> 
> TBH this type of changes doesn't need a review from a maintainer of the repo. Should we relax the branch restrictions for cases like this to speed up the dev cycle?

i'm ok with that. 👍 I'm not sure if there is a branch protection rule for these type of changes, but I think I have permissions to just bypass without reviewer if needed.

**Pull request merged**
